SCOPES.md

```diff
@@ -0,0 +1,70 @@
+# Prefixes
+Supported scope prefixes:
+* `project:releng:balrog`
+  * Used for Firefox/Fennec/Devedition style balrog tasks
+
+* `project:comm:thunderbird:releng:balrog`
+  * Used for Thunderbird style balrog tasks
+
+# Scopes
+* `{scope_prefix}:action:schedule`
+  * Tells balrogscript to Schedule a release to ship on balrog channel(s)
+  * **Conflicts**: with any other `{scope_prefix}:action:*`
+  * **Branch Restrictions**: None
+
+* `{scope_prefix}:action:submit-locale`
+  * Tells balrogscript to submit locale information for a release.
+    * This is **default** if no `action` specified in scopes.
+  * **Conflicts**: with any other `{scope_prefix}:action:*`
+  * **Branch Restrictions**: None
+
+* `{scope_prefix}:action:submit-toplevel`
+  * Tells balrogscript to submit the initial toplevel release information.
+  * **Conflicts**: with any other `{scope_prefix}:action:*`
+  * **Branch Restrictions**: None
```

Comment:
I don't think it's necessarily a branch restriction, but I don't think we have this type of scope defined for any of the nightly branches as this is specific to releases only.